go/libkb/globals.go

```diff
@@ -180,6 +183,13 @@ func (g *GlobalContext) Logout() error {
 	// get a clean LoginState:
 	g.createLoginStateLocked()
 
+	// remove stored secret
+	if g.SecretStoreAll != nil {
+		if err := g.SecretStoreAll.ClearSecret(username); err != nil {
```

Comment:
we're protecting the `SecretStoreAll` with a lock here but we don't in other places. Is that OK?